### PR TITLE
add mac os instructions

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -22,10 +22,20 @@ The script generates a [supervisord][] configuration file in
 This guide assumes a recent Debian/Ubuntu installation.
 
 * Install dependencies:
+    - Linux:
     ```shell
     sudo apt update
     sudo apt install python3 git supervisor curl
     ```
+    - Mac OS:
```
Comment (mgax) [inline Readme.md]: But at the top it still says:

> This guide assumes a recent Debian/Ubuntu installation.

Maybe rephrase that, and make separate sections with headlines for linux and mac os?

File: Readme.md [hunk 2]
```diff
@@ -39,16 +49,25 @@ This guide assumes a recent Debian/Ubuntu installation.
...
     ```shell
     ./cluster.py configure
     sudo ln -s $(pwd)/etc/supervisor-cluster.conf /etc/supervisor/conf.d/cluster.conf
     sudo supervisorctl update
     ```
 
+    - Mac OS:
+    ```shell
+    ./cluster.py configure
+    sudo mkdir /usr/local/etc/supervisor.d
```
Comment (mgax) [inline Readme.md]: AFAIK homebrew doesn't recommend using sudo on the `/usr/local` directory, so you shouldn't need root for this.

File: cluster.py [hunk 3]
```diff
@@ -174,21 +182,27 @@ class CONFIG:
 
 CONFIG.supervisor = lambda username: f'''\
 [program:consul]
+environment = PATH="{PATH.env}"
```
Comment (mgax) [inline cluster.py]: Actually, in [docker](https://github.com/liquidinvestigations/cluster/pull/22), I'd rather have all the logs redirect to supervisor's stderr, so I can watch all of them easily from outside docker. How do we reconcile what we need in a clean way? Let's move this to a separate PR.

File: examples/network-mac.sh [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+#       address = 10.66.60.01
+#
+#       [consul]
+#       address = 10.66.60.01
+
+bridge_name=liquid-bridge
+bridge_address=10.66.60.1
+public_interface=$(route get 8.8.8.8 | awk '/interface:/ {print $2}')
+
+ifconfig $public_interface alias $bridge_address/24 up
```
Comment (mgax) [inline examples/network-mac.sh]: I'm not sure what this command does, but it looks like something different from the Linux bridge logic. The whole point of having a bridge interface was to keep the cluster separate from the actual network (unless of course the admin configures traffic through e.g. iptables masquerading).

Can we find a Mac OS specific networking thing that is similar to the Linux bridge? Maybe a dummy interface?

File: examples/network-mac.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
```
Comment (mgax) [inline examples/network-mac.sh]: How about we keep all our code in `network.sh` and detect the architecture at runtime in one big `if`?

Comment (mugurrus) [reply examples/network-mac.sh]: I'd prefer to keep the scripts simple. There are separate instructions for Linux/Mac OS anyway.

File: cluster.py (hunk 3 above)
Comment (mugurrus) [reply cluster.py]: You can see them all with tail

File: examples/network-mac.sh (hunk 5 above)
Comment (mgax) [reply examples/network-mac.sh]: Ok let's keep them separate.

File: cluster.py (hunk 3 above)
Comment (mgax) [reply cluster.py]: They are in a docker container, you have to `docker exec` to get to them, and it's impractical to capture the logs from outside docker.

File: cluster.py [hunk 6]
```diff
@@ -174,21 +182,27 @@ class CONFIG:
...
 user = {username}
 command = {PATH.cluster_py} runserver consul
 redirect_stderr = true
+stdout_logfile = {PATH.log / 'consul.log'}
 autostart = {OPTIONS.supervisor_autostart}
 
 [program:vault]
+environment = PATH="{PATH.env}"
```
Comment (mgax) [inline cluster.py]: I forget, why are we changing `PATH`?

Comment (mugurrus) [reply cluster.py]: Without it it's unable to find `env`

File: cluster.py (hunk 3 above)
Comment (mgax) [reply cluster.py]: Ok, we can get the logs easily from `var/log`, let's do it.

File: cluster.py [hunk 7]
```diff
@@ -15,13 +15,16 @@
...
 from urllib.error import URLError
+import sys
 
 log = logging.getLogger(__name__)
 
 config = configparser.ConfigParser()
 config.read('cluster.ini')
 
 class PATH:
+    env = os.getenv('PATH')
```
Comment (gabriel-v) [inline cluster.py]: this is unused

File: cluster.py [hunk 8]
```diff
@@ -36,6 +39,7 @@ class PATH:
     supervisor_conf = etc / 'supervisor-cluster.conf'
 
     var = root / 'var'
+    log = var / 'log'
```
Comment (gabriel-v) [inline cluster.py]: this is also unused
